config/shopr.php

```diff
@@ -10,6 +10,14 @@
         'order-confirmation' => '',
     ],
 
+    /*
+     * Create a easier way to apply custom config upon base models.
+     */
+    'models' => [
+        'order' => null,
```

Comment:
I think I'd like to have the model keys reflecting the actual class names a bit more, as well as defaulting to the base models to clearify what type of value should be provided here. Something like this: 
```
'Order' => Happypixels\Shopr\Models\Order::class,
'OrderItem' => Happypixels\Shopr\Models\OrderItem::class,
```